SPECS/qemu-dp.spec

```diff
@@ -2,7 +2,7 @@ Summary: qemu-dp storage datapath
 Name: qemu-dp
 Epoch: 2
 Version: 2.10.2
-Release: 1.2.0
+Release: 1.2.1
```

Comment:
I'm not sure anymore about the release number increase, but let's keep it this way for now. I'll think something through later.